targets/python/source/source/PlayFabHTTP.py

```diff
@@ -3,28 +3,13 @@
 import PlayFabErrors
 import requests
 
-class PlayFabBaseObject():
-    pass
-
-# This is a base-class for all Api-request objects.
-# It is currently unfinished, but we will add result-specific properties,
-#   and add template where-conditions to make some code easier to follow
-class PlayFabRequestCommon(PlayFabBaseObject):
-    pass
-
-# This is a base-class for all Api-result objects.
-# It is currently unfinished, but we will add result-specific properties,
-# and add template where-conditions to make some code easier to follow
-class PlayFabResultCommon(PlayFabBaseObject):
-    pass
-
 class PlayFabJsonError():
-    def __init__(self):
+    def __init__(self, errorMessage):
```

Comment:
This feels insufficient.
Basically, the constructor param should take everything or nothing.
We don't want to lose any information.